src/compas/geometry/predicates/predicates_2.py

```diff
@@ -383,7 +386,7 @@ def is_polygon_in_polygon_xy(polygon1, polygon2):
         for i in range(len(polygon1)):
             line = [polygon1[-i], polygon1[-i - 1]]
             for j in range(len(polygon2)):
-                line_ = [polygon2[-j], polygon2[j - 1]]
+                line_ = [polygon2[-j], polygon2[-j - 1]]
```

Comment:
looking at this code now, would it not be much simpler (and more readable) to do the following:

```python
p1 = len(polygon1)
p2 = len(polygon2)
for i in range(-1, p1 - 1):
    line1 = polygon[i], polygon[i + 1]
    for j in range(-1, p2 - 1):
        line2 = polygon[j], polygon[j + 1]
```

Reply:
(see #1130 )